Automattic/jetpack
# Add product card for Jetpack anti-spam on anti-spam recommendation

#### Changes proposed in this Pull Request:

This PR adds a product card for Jetpack Anti-Spam on the recommendation step about spam management.

#### Jetpack product discussion


pbNhbs-1iw-p2#comment-7724

#### Does this pull request change what data or activity we track or use?


This PR does not add any new analytics events, but can make use of the following existing event to record clicks on the product card CTA button, passing the step slug as a property of the event:
```
jetpack_recommendations_step_product_card_button_click
```

#### Testing instructions:






* Set up your test site with this PR, make sure the site has a Jetpack Free plan.
* Navigate to: `/wp-admin/admin.php?page=jetpack#/recommendations/anti-spam`
* You should see the product card for Jetpack Anti-spam showing on the right side of the recommendation step:

![Screen Shot 2022-04-19 at 3 22 30 PM](https://user-images.githubusercontent.com/18016357/164080630-4bf44302-e5db-43ae-96e8-d48110fcd6da.png)

* The CTA button on the product card should link to a checkout page with the `Jetpack Anti-spam` product in the cart for the site that you are testing with.
* Repeat this test with a site that has a Jetpack Anti-spam product, Security plan or Complete plan - the product card should NOT show when viewing the anti-spam recommendation.
